editor/configs/emsColumns.tsx

```diff
@@ -79,17 +116,36 @@ export const ALL_EMS_COLUMNS = {
     valueRenderer: (value) => formatCrashMatchStatus(value.person_match_status),
     sortable: true,
   },
+  person_match_attributes: {
+    path: "person_match_attributes",
+    label: "Person match attributes",
+    valueRenderer: (record) => {
+      if (!record.person_match_attributes) return "";
+      return record.person_match_attributes
+        .map((attr) => formatMatchAttrName(attr))
+        .join("\n");
+    },
+    defaultHidden: true,
+  },
+  person_match_score: {
+    path: "person_match_score",
+    label: "Person match quality",
+    valueRenderer: (record) => formatMatchScore(record.person_match_score),
+    defaultHidden: true,
+  },
```

Comment:
Even though we're treating it as a binary high/low quality match at the moment, it seemed like a good idea to track this as a "score" in the database to leave open the possibility that users want finer control over this in the future.